cmd/snap-exec/main.go

```diff
@@ -137,6 +137,9 @@ func snapExecApp(snapApp, revision, command string, args []string) error {
 
 	// run the command
 	fullCmd := filepath.Join(app.Snap.MountDir(), cmd)
+	if command == "shell" {
+		fullCmd = "/bin/bash"
```

Comment:
from what I understand, we want this to be `os.Getenv("SHELL")`, defaulting to `/bin/bash`